src/containers/DeploymentsTableContainer/DeploymentsTableContainer.jsx

```diff
@@ -61,21 +60,34 @@ const DeploymentsTableContainer = (props) => {
     experiments = project.experiments;
   }
 
+  const [deployments, setDeployments] = useState([]);
+
+  const handleFetchAllDeploymentsRuns = async (isToShowLoader) => {
+    const deploymentsRuns = await handleFetchDeploymentsRuns(
+      projectId,
+      experiments,
+      isToShowLoader
+    );
+
+    setDeployments(deploymentsRuns);
+  }
+
   // HOOKS
   useEffect(() => {
     // fetching deployed experiments
-    handleFetchDeployedExperiments(projectId, experiments, true);
+    handleFetchAllDeploymentsRuns(true);
 
     // polling deployed experiments
     const polling = setInterval(
-      () => handleFetchDeployedExperiments(projectId, experiments, false),
+      () =>
+      handleFetchAllDeploymentsRuns(false),
       30000
     );
     return () => clearInterval(polling);
-  }, [experiments, handleFetchDeployedExperiments, projectId]);
+  }, [projectId]);
```

Comment:
Line 87:6:  React Hook useEffect has a missing dependency: 'handleFetchAllDeploymentsRuns'. Either include it or remove the dependency array  react-hooks/exhaustive-deps